tests/test_layout.py

```diff
@@ -155,6 +155,54 @@ def test_focus_wrap(hlwm, running_clients, running_clients_num):
             hlwm.call('focus down')
 
 
+@pytest.mark.parametrize("mode", ['setting', 'flag'])
+@pytest.mark.parametrize("setting_value", [True, False])
+@pytest.mark.parametrize("external_only", [True, False])
+@pytest.mark.parametrize("running_clients_num", [3])
```

Comment:
This confused me, what is the use of this parameter? And the name does not match the parameter in the list of the test function? Could be that I don't know enough about this fixture and this intended

Reply:
This parameter is implicitly used by the `running_clients` fixture and controls, how many clients are created. (`running_clients` is then a list of the defined size containing the clients). @dnnr implemented it this way back then if I recall correctly.

Reply:
Ok then. I just wondered because in the test_dump case below, both are used.
But seems to work, so closing this remark. Thanks for clearing it up!

Reply:
> Ok then. I just wondered because in the test_dump case below, both are used.
> But seems to work, so closing this remark. Thanks for clearing it up!

That's because I didn't know it before this PR that also works without explicitly mentioning `running_clients_num` in the function parameters.